tello_driver/CMakeLists.txt

```diff
@@ -29,44 +29,49 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 # Find packages
 find_package(ament_cmake REQUIRED)
 find_package(camera_calibration_parsers REQUIRED)
+find_package(class_loader REQUIRED)
```

Comment:
Is class_loader needed if we are not doing launch file composition? Is there any reason to add the RCLCPP_COMPONENTS_REGISTER_NODE macro if we are not doing launch file composition?

Reply:
Correct, class_loader and RCLCPP_COMPONENTS_REGISTER_NODE macro are only required for launch file and CLI (`ros2 component load ...`) composition.